From 05539bcedcc6beca8e56b913b209c75f75c28575 Mon Sep 17 00:00:00 2001
From: Xianzhong <xianzhong.li@nxp.com>
Date: Mon, 17 Apr 2017 11:17:42 +0800
Subject: [PATCH 1679/5242] MGS-2507 [imx7ulp] gpu clock is not aligned with
 design expectation

commit  8190f7f4337d5dc34898a2129ac85715c678b1be from
https://source.codeaurora.org/external/imx/linux-imx.git

Fixed the clock rate to 350M Hz to meet the design.

Signed-off-by: Yuchou Gan <yuchou.gan@nxp.com>
Date: 13 April, 2017
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx7ulp.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/imx/clk-imx7ulp.c b/drivers/clk/imx/clk-imx7ulp.c
index 61ff9ad..ab1403a 100644
--- a/drivers/clk/imx/clk-imx7ulp.c
+++ b/drivers/clk/imx/clk-imx7ulp.c
@@ -183,6 +183,7 @@ static void __init imx7ulp_clocks_init(struct device_node *scg_node)
 		imx_clk_prepare_enable(clks[clks_init_on[i]]);
         imx_clk_set_parent(clks[IMX7ULP_CLK_GPU2D], clks[IMX7ULP_CLK_APLL_PFD2]);
         imx_clk_set_parent(clks[IMX7ULP_CLK_GPU3D], clks[IMX7ULP_CLK_APLL_PFD2]);
+        imx_clk_set_rate(clks[IMX7ULP_CLK_APLL_PFD2], 350000000);
 
 	pr_info("i.MX7ULP clock tree init done.\n");
 }
-- 
1.7.9.5

